src/verify_cert.rs

```diff
@@ -163,6 +174,60 @@ impl<'a> ChainOptions<'a> {
     }
 }
 
+pub struct Path<'a> {
```

Comment:
WDYT about naming this something like `ValidatedPath` to emphasize that signatures, etc have been checked.

Also: I think it could use some rustdoc comments on the type + pub fns.